src/ngraph/node_output.cpp

```diff
@@ -15,21 +15,34 @@
 //*****************************************************************************
 
 #include "ngraph/node_output.hpp"
+#include "ngraph/log.hpp"
 #include "ngraph/node.hpp"
 #include "ngraph/op/get_output_element.hpp"
 
 namespace ngraph
 {
+    namespace
+    {
+        static bool remove_goe = true;
```

Comment:
Why do we need remove_goe as a global object? It can be a problem if user works with several nGraph functions.

Reply:
In normal (non-test) use, it is set once during initialization of a transformer that doesn't work when GetOutputElements are missing. To fix those transformers would take at least another month (they do need to be fixed).

Reply:
Could you add a comment that it is a temporary solution? And that it will be removed after the transformers fix.

Reply:
Can we plan the activity to fix those transformers in order to don't forget it?